prow/cmd/hook/main.go

```diff
@@ -96,6 +96,11 @@ func main() {
 	}
 	logrus.SetFormatter(logrusutil.NewDefaultFieldsFormatter(nil, logrus.Fields{"component": "hook"}))
 
+	if o.configPath == "" {
+		logrus.Warning("Hook requires explicitly setting --configPath to function correctly in production.")
+		logrus.Warning("--configPath will soon default to empty string.")
```

Comment:
Could you add a time to this please

Reply:
Could you suggest a time for this? Also, should I undo the changes for the default and add TODOs for the time being?

Reply:
yes, we should not change the defaults in this PR. 6months is a good time frame to sunset this default